### PR TITLE
#3921: Workaround for MethodAccessorImpl NoClassDefError

### PR DESCRIPTION
**Pull request by xerial**

### CODE REVIEW

File: sbt-plugin/src/main/scala/org/scalajs/sbtplugin/LinkerImpl.scala [hunk 1]
```diff
@@ -75,6 +75,7 @@ object LinkerImpl {
     private val parentPrefixes = List(
       "java.",
       "scala.",
+      "sun.",
```
Comment (sjrd) [inline sbt-plugin/src/main/scala/org/scalajs/sbtplugin/LinkerImpl.scala]: We'll need a comment explaining why we have this, with a reference to the issue and possibly also the OpenJDK issue.

Comment (xerial) [reply sbt-plugin/src/main/scala/org/scalajs/sbtplugin/LinkerImpl.scala]: ok. I'll add

**Comment by sjrd** [comment]
Also this PR must target the `release-1.0.0` branch. The `master` branch is now for 1.0.1 and later.

**Comment by xerial** [comment]
ok. I'll retarget the branch. May need to create another PR.

**Comment by xerial** [comment]
Created another PR #3928 to target release-1.0.0 branch.

**Pull request closed without merging**